config/test.js

```diff
@@ -5,8 +5,8 @@ module.exports = {
 		name: 'hah-test'
 	},
 	facebook: {
-		appId: process.env.FACEBOOK_APP_ID,
-		appSecret: process.env.FACEBOOK_APP_SECRET,
+		appId: process.env.V_FACEBOOK_APP_ID,
+		appSecret: process.env.V_FACEBOOK_APP_SECRET,
```

Comment:
That was to distinguish between the Verification portal Facebook app, and any other Facebook app we might need, but it's looking now like we won't need another app, so it's probably unnecessary.